shared/video-sdk/authentication-workflow/reference/android.mdx

```diff
@@ -1,9 +1,17 @@
 <PlatformWrapper platform="android">
-        
+
 ### API reference
 
+<ProductWrapper notAllowed="media-accelerator">      
```

Comment:
We need to track down the API ref files.

Reply:
I found API references on the Chinese website:
https://docs.agora.io/cn/RTSA/API%20Reference/rtsa_c/index.html
https://docs.agora.io/cn/RTSA/API%20Reference/rtsa_java/classio_1_1agora_1_1rtc_1_1_agora_rtc_service.html